terminal/safety/safety.py

```
@@ -141,4 +141,34 @@ def validate_confirmation(self, confirmation: str, risk_level: RiskLevel) -> boo
             return confirmation.lower() in ["y", "yes"]
         else:
             return confirmation.lower() in ["y", "yes"]
+
+def check_command_safety(cmd: str) -> bool:
+    safety_result = CommandSafety().analyse_command(cmd)
+    
+    if safety_result.blocked and safety_result.risk_level == RiskLevel.CRITICAL:
+        print(f"[red]{safety_result.warning[0]}[/red]")
+        if safety_result.suggestions:
+            print(f"[yellow]Suggestions:[/yellow]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)